src/styles/style.css

```diff
@@ -1,3 +1,15 @@
 .ka-tbody .ka-tr {
-  border-bottom: 2px solid #ececec; /* Red border bottom */
+  border-bottom: 2px solid #ececec;
+}
+
+.ka-thead-cell {
+  padding: 15px 20px;
+  color: #ffffff;
+  background: #8d8c8c;
+}
+
+.ka-thead-background {
+  z-index: 1;
+  background: #a8a6a6;
```

Comment:
@upendraTekdi use colors from the theme variables

Reply:
removed this unwanted style file  and add some css in global css file where we can not get theme colors.